curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4b2.md

```diff
@@ -0,0 +1,141 @@
+---
+id: 66c8ba975ee7230e29f6c4b2
+title: Step 7
+challengeType: 1
+dashedName: step-7
+---
+
+# --description--
+
+Now that you're done with the `getLoanMessage` function, you should test it out against some `annualIncome` and `creditScore` instances. Remember those are the parameters the function takes.
+
+Use the table below to create 4 variables and their values:
+
+| Variable Name | Value   |
+| ----------- | ------- |
+| `duplexLoanMsg`       | `getLoanMessage(85000, 850)` |
+| `condoLoanMsg`   | `getLoanMessage(65000, 690)`   |
+| `carLoanMsg`   | `getLoanMessage(45000, 660)`   |
+| `noLoanMsg`   | `getLoanMessage(25000, 550)`   |
+
+After that, log each variable to the console to see the messages.
+
+With that, your loan qualification checker project is completed.
+
+# --hints--
+
+You should create a `duplexLoanMsg` variable set to `getLoanMessage(85000, 850)`.
+
+```js
+assert.match(code, /(const|let|var)\s+duplexLoanMsg\s*=\s*getLoanMessage\(85000,\s*850\)/);
+```
```

Comment:
we don't need a regex here.
we should use the built in assert methods instead

````suggestion
You should have a variable called `duplexLoanMsg`.

```js
assert.isNotNull(duplexLoanMsg);
```

Your `duplexLoanMsg` variable should be set to the result of `getLoanMessage(85000, 850)`.

```js
assert.equal(duplexLoanMsg, getLoanMessage(85000, 850));
```
````